subprojects/docs/src/docs/userguide/announce_plugin.adoc

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 [[announce_plugin]]
-== The Announce Plugin
+= The Announce Plugin
 
 The Gradle announce plugin allows you to send custom announcements during a build. The following notification systems are supported:
 
```

Comment:
One minor observation is that these links now show up in italics - not sure if that's expected or not.

Reply:
Only links to other chapters should be italic. I'll fix this.